apps/prairielearn/elements/pl-external-grader-results/pl-external-grader-results.py

```diff
@@ -39,7 +39,7 @@ def ansi_to_html(output: str | None) -> str | None:
         return None
     try:
         return conv.convert(output, full=False)
-    except Exception as exc:
+    except (ValueError, RuntimeError) as exc:
```

Comment:
these should be the only two types of exceptions raised now.

Reply:
What guarantees do we have of that, or that this won't change in the future? When would we not want to handle _any_ exception here?

Reply:
Resolved with additional clause in https://github.com/PrairieLearn/PrairieLearn/pull/11104/files#diff-4345d44c5dc72eccdfedbce8ea8a53bf1157725a5bd9a98242afd9ed1a3393deR45.

Reply:
Reverted in https://github.com/PrairieLearn/PrairieLearn/commit/55f351fea4dc4304cdede5f9a3f902b484f6ebf7.